test/overlay-dirs.bats

```diff
@@ -53,6 +53,32 @@ EOF
     stacker build
 }
 
+@test "build binary and import in distroless" {
+    mkdir dir_to_overlay
+    chmod -R 777 dir_to_overlay
+    cat > stacker.yaml << EOF
+first:
+    from:
+        type: oci
+        url: $CENTOS_OCI
+    binds:
+        - dir_to_overlay -> /dir_to_overlay
+    run: |
+        touch /dir_to_overlay/helloworld.txt
+    build_only: true
+justbinary:
+    from:
+        type: docker
+        url: docker://gcr.io/distroless/base
+    overlay_dirs:
+        - source: dir_to_overlay
+          dest: /dir_to_overlay
+EOF
+    stacker build
+    ls dir_to_overlay | grep "helloworld.txt"
```

Comment:
What we want to verify is the `justbinary` image contains `helloworld.txt`.
Please unpack `justbinary` using skopeo and verify the file is there.

Reply:
Added, + documentation